src/adera/activescene/VehicleBuilder.cpp

```diff
@@ -101,7 +103,7 @@ void VehicleBuilder::index_prefabs()
             continue; // No prefab data
         }
 
-        for (osp::PrefabId j = 0; j < pPrefabData->m_prefabNames.size(); ++j)
+        for (osp::PrefabId const j : std::views::iota(0u, pPrefabData->m_prefabNames.size()))
```

Comment:
I don't believe there's anything wrong with the original for loop. It's more modern? but no improvements in type safety or anything, slightly longer compile times, and probably confuses new contributes or less experienced programmers.

I'd avoid the ranges if other parts of the library are not being used (and yes I probably have a bit of a grudge against it as it made of my other projects a huge headache, and also Rust exists). I feel that ranges is something we should give a few more years to mature before using seriously.

Also noting that many of these identifiers are less of 'numbers', and are more of 'identifiers with the special property of being sequential'. PrefabId is planned to eventually be changed to StrongId<...>, so iota probably wouldn't work.